js/counter.js

```diff
@@ -0,0 +1,23 @@
+const endDate = new Date("Apr 8, 2018 23:59:59").getTime(); // The date is set to the end of a final sprint
```

Comment:
Do not use comment to describe what const is up to. Better use a longer name or specified name. Use:
const premiereDate or premiereDataAsOfFinalSprint instead of using // description to explain what is this const about :) Less comments - better selfdescribed script :)